crates/typst/src/math/matrix.rs

```diff
@@ -29,7 +30,8 @@ const DEFAULT_STROKE_THICKNESS: Em = Em::new(0.05);
 
 /// A column vector.
 ///
-/// Content in the vector's elements can be aligned with the `&` symbol.
+/// Content in the vector's elements can be aligned with the
+/// [`align`]($category/math/vec.align) parameter, or the `&` symbol.
```

Comment:
```suggestion
/// [`align`]($math.vec.align) parameter, or the `&` symbol.
```
_should_ work in principle. did you not try it or did it not work?

Reply:
Oh no I didn't try it. I just had no idea which worked and when I looked for some examples I could only find ones with `$category` prepended. I'll change it and try it after I finish with the stretch PR.